src/quotamodel.c

```diff
@@ -1249,13 +1253,10 @@ flush_to_table_size(void)
 		}
 	}
 
-	if (delete_entries_num) delete_from_table_size_map(delete_statement.data);
-	if (insert_entries_num) insert_into_table_size_map(insert_statement.data);
+	if (delete.tableids) delete_from_table_size_map(delete.tableids, delete.segids);
+	if (update.tableids) update_table_size_map(update.tableids, update.sizes, update.segids);
```

Comment:
> If we touch this code anyway, we could use pgindent-style formatting as well.

No!!! This is a `diskquota` and it has its own formatter, which checks the code with a separate pipeline!

Reply:
I meant something like this would read much easier and allow to set a breakpoint in the debugger:
```
	if (delete.tableids)
	{
		delete_from_table_size_map(&delete);
	}
	if (update.tableids)
	{
		update_table_size_map(&update);
	}
```
Such a formatting already used few lines above.

Reply:
added